src/Http/Controllers/SettingController.php

```diff
@@ -50,6 +51,36 @@ public function updateSlack(Request $request)
             $validated_data['slack_integration_default_channel'],
         ], true);
 
+        setting([
```

Comment:
At this stage, it would be better to spawn a validation class and update the method signature.
You'll be able to check all form inputs, types and still beeing able to call value like an object.
- https://laravel.com/docs/6.x/validation#form-request-validation

You'll find implementation sample in core :
- https://github.com/eveseat/web/blob/master/src/Http/Validation/SeatSettings.php
- https://github.com/eveseat/web/blob/master/src/Http/Controllers/Configuration/SeatController.php#L86-L94

That will also help you to reduce method complexity.